client/client.go

```diff
@@ -2,12 +2,20 @@ package main
 
 import (
 	"context"
+	"errors"
 	"flag"
 	"time"
 
-	"k8s.io/klog/v2"
-	"google.golang.org/grpc"
 	pb "github.com/BetterGR/course-microservice/course_protobuf"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"k8s.io/klog/v2"
+)
+
+// Predefined static errors.
+var (
+	errCreateCourse = errors.New("failed to create course")
```

Comment:
This is a bad practice. It's okay since this is an example code file that will not be actually ran.